### PR TITLE
consider packages and data-managers as well

### PR DESCRIPTION
**Pull request by bgruening**

### CODE REVIEW

**Comment by bgruening** [comment]
ping @nsoranzo @mvdbeek

**Comment by nsoranzo** [comment]
Should we skip `planemo test` for packages and data managers?

**Comment by bgruening** [comment]
This would at least save a few seconds I guess.

**Comment by yhoogstrate** [comment]
Is this just for linting or is `planemo test` now capable of testing data managers?

**Comment by nsoranzo** [comment]
As I've mentioned above, `planemo test` does not work for data managers yet and will never work on packages, so it should probably be skipped. This is for linting and especially for automatic upload.

**Comment by bgruening** [comment]
@nsoranzo can we merge this then or try to skip the planemo test before?

**Comment by bgruening** [comment]
@nsoranzo I have added the skip, the failing test is due to and ssl certification error - I hope they will fix it. Can we merge?

File: tools/fastqc/rgFastQC.xml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<tool id="fastqc" name="FastQC" version="0.68">
+<tool id="fastqc" name="FastQC" version="0.67">
```
Comment (nsoranzo) [inline tools/fastqc/rgFastQC.xml]: That should be reverted.

Comment (bgruening) [inline tools/fastqc/rgFastQC.xml]: Done.

**Comment by nsoranzo** [comment]
> the failing test is due to an SSL certification error - I hope they will fix it.

That's strange, the certificate validates fine with both Firefox and Planemo on my laptop, is it a Travis problem?

**Comment by bgruening** [comment]
It can be a travis problem they updated the image last week or so.

**Comment by bgruening** [comment]
:) Thanks.

**Pull request merged**